evm/src/cpu/kernel/asm/hash/sha2/main.asm

```diff
@@ -33,21 +31,19 @@ global sha2_pad:
     DUP1
     // stack: num_bytes, num_bytes, retdest
     %add_const(8)
-    %div_const(64)
+    %shr_const(6)
```

Comment:
Nit: `shr_const(6)` is actually very marginally less efficient than `div_const(64)` because it needs an extra memory lookup.

That said, the difference is so small that I'd just go with whatever makes for cleaner code.

Reply:
Ah interesting, thanks! I'll actually change a few the other way then